ONSdigital/ras-rm-docker-dev
# Add respondent home ui

# Motivation and Context

This adds the respondent-home-ui application. The image in DockerHub is currently based on the Dockerfile from this branch: https://github.com/ONSdigital/respondent-home-ui/blob/ssd/Dockerfile.


# What has changed



* `.env` - Had to base64 encode multiline keys to workaround newlines with docker compose reading env vars from file, see [here](https://goo.gl/GXqLCv). I could have used the multiline YAML inline in `rm-services.yml` but I tried to keep everything consistent with the other apps. 
* `rm-services.yml` - Added new app and env vars.

# How to test?



* Run the following:
```bash
docker-compose -f rm-services.yml up -d respondent-home-ui
```
* Open a browser and navigate to [localhost:9292](localhost:9292)
* Check that the UI loads correctly.

# Links



[Trello](https://trello.com/c/SwkUMf9O/99-dev-task-7-ext-access-eq-social-survey-rh-8)
